stores/sql_test.go

```diff
@@ -217,11 +260,13 @@ func (s *SQLStore) contractsCount() (cnt int64, err error) {
 func (s *SQLStore) overrideSlabHealth(objectID string, health float64) (err error) {
 	err = s.db.Exec(fmt.Sprintf(`
 	UPDATE slabs SET health = %v WHERE id IN (
-		SELECT sla.id
-		FROM objects o
-		INNER JOIN slices sli ON o.id = sli.db_object_id
-		INNER JOIN slabs sla ON sli.db_slab_id = sla.id
-		WHERE o.object_id = "%s"
+		SELECT * FROM (
```

Comment:
Is there a reason for the new subquery?

Reply:
If you run it on MySQL it fails. I didn't bother adding an `isSQLite` check because it's testing.